Bottom: fbb689bc3e5fc4dc670b312a486301677a0617aa
Top:    66702d5a96597802b71c185c76857f89d0ac0c82
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 13:47:01 +0800

Refresh of autofs-5.0.7-add-dumpmap-to-file.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index a78428a..9704ceb 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1933,7 +1933,7 @@ int main(int argc, char *argv[])
 	int logpri = -1;
 	unsigned ghost, logging, daemon_check;
 	unsigned dumpmaps, foreground, have_global_options;
-	const char *dumpnames;
+	const char *dumpnames = NULL;
 	time_t timeout;
 	time_t age = time(NULL);
 	struct rlimit rlim;
